src/core.ts

```diff
@@ -380,6 +380,8 @@ export const TrackingConsentDetails = t.intersection([
     metadata: t.unknown,
     /** When the metadata was last updated */
     metadataTimestamp: t.string,
+    /** When the consent data was last modified */
+    lastModified: t.string,
```

Comment:
Interesting. Seems like this necessitates a change to the backend. I'm not sure I follow how `lastModified` could differ from `timestamp` (apart from being `null` when `confirmed: false`)

Reply:
timestamp refers also to when we resolve the timestamp

So if you land on a page and haven't made any consent preferences, timestamp would be auto-resolved to the current timestamp when you land on a page, whereas lastModified would be null. When you explicitly update/opt-in/opt-out, when would update lastModified.

Reply:
> When you explicitly update/opt-in/opt-out, when would update lastModified.
...but not timestamp?

Reply:
@MadDataScience we need to modify the backend codec? or what you thinking? Can you help track that work in Height?

Reply:
> When you explicitly update/opt-in/opt-out, when would update lastModified.

We would update both @MadDataScience 

So timestamp includes BOTh when we auto-resolve AND when the user has opted in or out. However, for the purposes of syncing, we also care about knowing when they've explicitly taken action versus just when we last-resolved it for them.

Reply:
I want to make sure I understand what's actually going on. I don't understand why timestamp wouldn't be updated when you explicitly opt-in/out

Reply:
it would be, but it's just too lossy for the purpose of sync, because we can't tell if the timestamp means the user made a choice or if it's just airgap auto-resolving

Reply:
So `lastModified = confirmed && timestamp`? That seems redundant